doc/sphinx/user-extensions/syntax-extensions.rst

```diff
@@ -462,6 +462,62 @@ Displaying information about notations
 
    :flag:`Printing All` to disable other elements in addition to notations.
 
+.. cmd:: Print Notation @string
+
+   Displays information about the previously reserved notation string
+   :token:`string`, namely the level at which it was reserved and its
+   associativity.
+
+   .. example::
+
+      .. coqtop:: all
+
+         Reserved Notation "x $ y" (at level 123, right associativity).
+         Print Notation "_ $ _".
+
+   Variables can be indicated with either "_" or names, as long as these can not
+   be confused with notation symbols. When confusion may arise, for example with
+   notation symbols that are entirely made up of letters, use single quotes to
+   delimit those symbols.
+
+   .. example::
+
+      .. coqtop:: all
+
+         Reserved Notation "x 'mod' y" (at level 40, no associativity).
+         Print Notation "_ mod _".
+         Print Notation "x 'mod' y".
+
+   There must always be (at least) a space between notation symbols and
+   arguments, even when the notation format does not include those spaces.
+
+   .. example::
```

Comment:
If this is just a property of notation strings, maybe should just be a "Note:" here.  Also we try to keep all the important text together before getting into examples to make it quicker to read the text.  You can hyperlink from the text to the example if appropriate, but these are probably so close it's not needed.  Also we like to describe examples as in the `:: example:: my description`

Reply:
Ok, thanks! I have changed the doc accordingly.